src/puppetlabs/puppetdb/jdbc.clj

```diff
@@ -647,3 +654,46 @@
     (str "in ("
          (str/join "," (repeat (count coll) inner))
          ")")))
+
+(defn current-user []
+  (-> (query-to-vec "select user") first :user))
```

Comment:
I'm getting _[{:current_user "pdb_test_migrator"}]_ as the result for this "select user" query locally. May need to change _:user_ to _:current_user_ here.

Reply:
Travis looks like it got _:user_ for the column name in the result so maybe it's due to a diff version of PG or something. I'm seeing _current_user_ be returned as the column name for the query _"select user"_ on n2 as well. Maybe we could use the query _"select current_user"_ to avoid the column name switching.